### ISSUE TITLE
Kubelet is not allowed to use privileged PSP

### ISSUE THREAD

**Issue by invidian** [kinvolk/lokomotive#50]
And because of this, following error is printed:
```
E0226 10:39:09.237346   12034 kubelet.go:1662] Failed creating a mirror pod for "kube-apiserver-6f5b85f5db-6n2q8-ip-10-0-2-169_kube-system(8122a931b691267d810b16f3f2435a8d)": pods "kube-apiserver-6f5b85f5db-6n2q8-ip-10-0-2-169" is forbidden: unable to validate against any pod security policy: [spec.volumes[1].hostPath.pathPrefix: Invalid value: "/usr/share/ca-certificates": is not allowed to be used spec.volumes[2]: Invalid value: "emptyDir": emptyDir volumes are not allowed to be used spec.securityContext.hostNetwork: Invalid value: true: Host network is not allowed to be used spec.volumes[0]: Invalid value: "hostPath": hostPath volumes are not allowed to be used spec.volumes[1]: Invalid value: "hostPath": hostPath volumes are not allowed to be used spec.volumes[3]: Invalid value: "hostPath": hostPath volumes are not allowed to be used]
```

We already allow `kubelet` to use `pod-checkpointer` PSP here: https://github.com/kinvolk/lokomotive/blob/master/assets/lokomotive-kubernetes/bootkube/resources/charts/kubernetes/templates/kubelet-pod-checkpointer-psp-role-binding.yaml. Perhaps we should do the same for `privileged` policy (or whatever PSP is used for `kube-apiserver`.

**Comment by surajssd** [comment]
The title says Kubelet is not allowed the PSP, but the error says there is no matching PSP for bootstrap apiserver. I am confused by the description. :thinking:

**Comment by invidian** [comment]
@surajssd errors does not mention "bootstrap" anywhere :wink:

**Comment by surajssd** [comment]
@invidian yes it is not mentioned because the bootstrap is an abstract concept. I concluded that it is bootstrap apisever because the node name at the end of the pod name is only typical of static pods and we run our bootstrap control plane as static pods.

**Comment by invidian** [comment]
In any case, the solution is here:
>We already allow kubelet to use pod-checkpointer PSP here: https://github.com/kinvolk/lokomotive/blob/master/assets/charts/control-plane/kubernetes/templates/kubelet-pod-checkpointer-psp-role-binding.yaml. Perhaps we should do the same for privileged policy (or whatever PSP is used for kube-apiserver.

Kubelet is creating a static Pod in the API to be able to represent it etc, so it needs to be allowed to use a matching PSP, otherwise this will fail.
